packages/core/src/lib/managers/keyboard.manager.ts

```diff
@@ -0,0 +1,33 @@
+import { CopyAction } from '../actions/copy.action';
+import { DeleteKeyPressed } from '../actions/delete-key-pressed.action';
+import { PasteAction } from '../actions/paste.action';
+import { DiagramEngineCore } from '../engine.core';
+import { NodeModel, PointModel, PortModel, LinkModel } from '../models';
+
+export class KeyboardManager {
+  protected engine: DiagramEngineCore;
+  clipboard: (NodeModel | PointModel | PortModel | LinkModel)[]
+
+  constructor(_diagramEngine: DiagramEngineCore) {
+    this.engine = _diagramEngine;
+    this.clipboard = []
+  }
+
+  onKeyDown(event: KeyboardEvent) {
+    if (event.keyCode === 8 || event.keyCode === 46) {
```

Comment:
what if the user want to override this specific default keys?
and use key combination instead?
what if he would want to show some sort of a dialog prior to deleting?